src/tests/test_cases/sections.py

```diff
@@ -274,6 +274,23 @@ def missing_colon_google_style_section():  # noqa: D406, D407
     """
 
 
+@expect("D417: Missing argument descriptions in the docstring "
+        "(argument(s) y are missing descriptions in "
+        "'bar' docstring)", func_name="bar")
+def _test_nested_functions():
+    x = 1
+
+    def bar(x=1, y=2):  # noqa: D207, D213, D406, D407
```

Comment:
Remove `x` from `bar`'s parameters so that `bar` will be use closure.

Reply:
Done.